### ISSUE TITLE
maps

### ISSUE THREAD

**Issue by wzb-ipi** [wzb-ipi/dashdash#25]
pull maps from map package instead of requiring shape files

@juliosolisar were you thinking of looking at that? others?

**Comment by juliosolisar** [comment]
Yes. But I am super worried about different IDs between shapefiles and data not merging. What's your take on that? How do you imagine it working?

**Comment by juliosolisar** [comment]
I actually closed the issue for this reason 😅

**Comment by wzb-ipi** [comment]
I imagine
1. people provide map and level and name of the ID
2. we check that they do indeed match?

**Comment by juliosolisar** [comment]
Ok! I guess warning them and working with whatever matches?

**Comment by macartan** [comment]
yes, warning I think
maybe a statement below map saying which areas are there that shouldn't be
nd which are not there that should be

**Comment by juliosolisar** [comment]
will work on that and get back to you soon
